webview-ui/src/components/chat/ChatRow.tsx

```diff
@@ -453,7 +455,7 @@ export const ChatRowContent = ({
 									style={{
 										color: normalColor,
 										fontWeight: "bold",
-									}}>{`API Request (Retrying failed attempt ${retryStatus.attempt}/${retryOperations})...`}</span>
+									}}>{`API Request (Retrying failed attempt ${retryStatus.attempt}/${retryOperations}.${formatDelay(retryStatus.delaySec)})...`}</span>
```

Comment:
Can you have something simpler like `{`API Request (Retry ${retryStatus.attempt}/${retryOperations} in ${formatDelay(retryStatus.delaySec)})`}`